h5p-editor-bildetema-words-topic-image/src/components/editor/Editor.tsx

```diff
@@ -109,6 +113,35 @@ export const Editor: React.FC<EditorProps> = ({
     setHotspots(updatedHotspots);
   };
 
+  const handleCircleDrag = (pointUpdate: PointUpdate): Point => {
+    if (!ref.current) {
+      return pointUpdate.from;
+    }
+
+    const rect = ref.current.getBoundingClientRect();
```

Comment:
We might want to avoid running `getBoundingClientRect` on every frame. The method is a usual culprit which makes apps appear slow. Is there a way we can cache the element's size or calculate it without triggering layout?

Reply:
todos added